### ISSUE TITLE
Problem with Port 8080

### ISSUE THREAD

**Issue by Jeern** [jbogard/Respawn#28]
Hi, I have started work on the Firebird Adapter as we talked abut. But I run into problems already. First I want to test your docker-compose command to see if I can add Firebird via docker if it has been containerized. The docker-compose.yml references a tool called adminer on port 8080 - this is the standard port for RavenDB which I have installed. So I cannot run docker-compose up -d it fails on the adminer part. I don't know if this has to be 8080 ? If I change the port to 8079 I can run the docker-compose command. But I cannot access localhost:8079 in the browser like I expected.

**Comment by Jeern** [comment]
Oh well. Perhaps this is kind of a non-issue - I can just stop the RavenDB service while working on this. But I wonder why the Adminer service cannot be configured to other ports than 8080.  They must have misunderstood what configuring is all about and hardcoded 8080 in their system :)

**Comment by jbogard** [comment]
You can change that in the docket compose file.

**Comment by Jeern** [comment]
I know. You misunderstand me. That is what I did. But it did not work. I had to change it back to 8080 and stop the Raven service to make it work. Could be that they have hardcoded something, or perhaps the service got into an inconsistent State when I first ran it. Anyway the issue can be closed. BTW I could not figure out what to enter in the adminer tool to access the databases (postgres and mysql). Seemed obvious - I tried localhost as server, User / Password from docker-compose file, and database = respawn_[something] which I got from running the docker-compose command. I also tried at lot of other combinations.
